src/en/models-cmr.md

```diff
@@ -113,21 +124,23 @@ These are applied similarly to how standard model access is applied, via the
 `juju grant|revoke <user> <access level> <offer url>`
 
 Revoking a user's consume access will result in all relations for that user to
-that offer to be suspended. If the consume access is granted anew, each relation
-will need to be individually resumed. Suspending and resuming relations are
-explained in more detail later.
+that offer to be suspended. If the consume access is granted anew, each
+relation will need to be individually resumed. Suspending and resuming
+relations are explained in more detail later.
 
-## Relating to offers
+### Relating to offers
 
 If a user has consume access to an offer, they can deploy an application in
 their model and establish a relation to the offer by way of its URL.
 The controller part of the URL is optional if the other model resides in
 the same controller.
 
-`juju relate <application> <offer url>`
+`juju relate <application>[:<application endpoint>] <offer url>[:<offer endpoint>]`
```

Comment:
add-relation

I suppose in the sense that offers are external objects, doing things this way around *feels* correct. I wonder if it would then make sense to change every other add-relation command in the docs

Reply:
I'm not sure how it is more correct by using `relate`. I would like to stay consistent by not using aliases. Maybe I'm missing your point.

Reply:
I meant the order of the application / offer url. Normally we do `add-relation <provider> <consumer>` 

we should definitely use add-relation

Reply:
Ohh